lib/service/cfg.go

```diff
@@ -524,6 +524,12 @@ type ProxyConfig struct {
 
 	// DisableALPNSNIListener allows turning off the ALPN Proxy listener. Used in tests.
 	DisableALPNSNIListener bool
+
+	UiConfig UiConfig
```

Comment:
```suggestion
	UI UiConfig
```

Similarly, `UI` is sufficient here. It's in a config struct, so we don't need to suffix every field name with `Config`.